src/main/java/mate/academy/dao/impl/MovieDaoImpl.java

```diff
@@ -45,6 +47,11 @@ public Optional<Movie> get(Long id) {
```

Comment:
After throwing DataProcessingException, it's necessary to roll back transaction. Add transaction.rollback() before throwing exception.